polus-vector-label-plugin/src/mask.py

```diff
@@ -0,0 +1,580 @@
+import numpy as np
```

Comment:
If this code is from cellpose, please include a link to their repository to give them credit.